### PR TITLE
docs: Update the doc for minikube installing kata

### PR DESCRIPTION
**Pull request by YchauWang**
The command for intalling kata in minikube still keeping the old path of
the packaging project in the 1.x branch. This commit changed the path to
2.0-dev branch.

Fixes: #619

Signed-off-by: Ychau Wang <wangyongchao.bj@inspur.com>

### CODE REVIEW

File: docs/install/minikube-installation-guide.md [hunk 1]
```diff
@@ -126,10 +126,10 @@ Containers components to help with this, and then use `kubectl` on the host (tha
...
 
 ```sh
-$ git clone https://github.com/kata-containers/packaging.git
-$ cd packaging/kata-deploy
-$ kubectl apply -f kata-rbac.yaml
-$ kubectl apply -f kata-deploy.yaml
+$ git clone https://github.com/kata-containers/kata-containers.git
+$ cd kata-containers/tools/packaging/kata-deploy
+$ kubectl apply -f kata-rbac/base/kata-rbac.yaml
+$ kubectl apply -f kata-deploy/base/kata-deploy.yaml
```
Comment (jodh-intel) [inline docs/install/minikube-installation-guide.md:132]: Looks like these 2 lines need to be back-ported to https://github.com/kata-containers/documentation/.

**Review by jodh-intel — APPROVED** [review]
Thanks @YchauWang.

lgtm

File: docs/install/minikube-installation-guide.md (hunk 1 above)
Comment (YchauWang) [reply docs/install/minikube-installation-guide.md:132]: yes, the 1.x doc project has the same error. Thank you @jodh-intel. I am sorry just force on 2.0-dev.

**Comment by YchauWang** [comment]
@jodh-intel the commit message check always report the "Failed to find subsystem in subject". Is it the "docs:" subsystem error, and what the write subsystem?

**Review by GabyCT — APPROVED** [review]

**Pull request merged**